src/base.config.ts

```diff
@@ -153,10 +155,46 @@ function loadRoutingOutlets() {
 	return outlets;
 }
 
+const shimHasFlags: { [index: string]: true } = {
+	'dom-webanimation': true,
+	'dom-intersection-observer': true,
+	'dom-resize-observer': true
+};
+
+class HasDojoShimPlugin {
+	apply(compiler: webpack.Compiler) {
+		compiler.plugin('compilation', (compilation: any) => {
+			compilation.plugin('seal', function() {
+				Object.keys(compilation._modules).forEach((key) => {
+					const module = compilation._modules[key];
+					if (
+						module.issuer &&
+						!/@dojo(\/|\\)cli-build-app(\/|\\)bootstrap.js$/.test(module.issuer.userRequest)
+					) {
+						if (/@dojo(\/|\\)framework(\/|\\)shim(\/|\\)WebAnimations\.(m)?js$/.test(module.userRequest)) {
```

Comment:
would be good if we could just write a loop that does this for many. and hopefully without a regex? it's a bit messy at the moment

Reply:
I originally started doing it in a loop but fell back to just being explicit for the time being as we only have 3 at the moment. I will take another look.

the regex is tough as it's a path so need to support the window/unix separators, Again will take another pass.

Reply:
you should be able to just make it the appropriate based on os right? then it can go back to basically a string other than the extension?

Reply:
ie use `path.normalize`?

Reply:
I used `path.sep`